source/process/spec-process.rst

```diff
@@ -0,0 +1,61 @@
+Spec Review Process
+===================
+
+This is a rough guideline for drafting and reviewing specs. It’s meant to be used as a starting point, and not all specs will follow this exact process - please rely on your judgment, as not all steps may apply.
+
+1. **Brainstorm / research.**
+
+    - Research topic
+    - Research customer requirements
+    - Pair design brainstorm
+    - User workflows
+    - Consider options / vision for UI
+    
+2. **Initial draft.**
+
+    - Goal / “Why"
+    
+      - Link to customer data / feedback
+      - Link back to company objectives
+      
+    - User Stories
+    - Rough and Quick Wireframes / Mock ups
+    
+      - Flows of screens
+      - Focus should be on the mock ups, with text that supports design
+      
+    - Questions that still need to be addressed
+    - What is “Not in scope” for the spec (things that are purposefully left out, to be addressed at another time)
+    - Success Metrics
+    - Pair reviews
+    
+      - Iterate on designs as needed
+      - Early check in with developers to identify any potential issues
+      
+3. **Share with team.**
+
+    - Post draft with @channel in Spec Review channel, asking to review and add comments
+    
+4. **Spec review.**
+
+    - Schedule a meeting with interested people to discuss comments and any other questions about the spec, try to include at least one member from each of the following teams:
+    
+      - UX Design
+      - PM
+      - Dev
+      - Test
+      
+5. **Update draft.**
+
+    - Update based on feedback, including wireframes / mock ups
+    
+6. **Review again.**
+
+    - Post updated draft in Spec Review channel, asking the people who previously commented for another review
+    - Share with interested customers for feedback
+    
+7. **Repeat reviews/updates.**
+
+    - Repeat as needed until final draft is signed off on by UX Design, PM, Dev, Test
+    
+8. **Break into tickets with dev.**
```

Comment:
I'm going to leave as is, because I think we're still figuring this out. Likely the dev will break it out into tickets, but I want PM involved as well to make sure they understand the tickets, that the tickets are a reasonable size, and that larger features are being worked on in a way that will make it easy to spin up a test server at the earliest stage possible, so we avoid having features "go dark".